kinetic/distribution.yaml

```diff
@@ -14907,6 +14907,31 @@ repositories:
       url: https://github.com/SvenzvaRobotics/svenzva_ros.git
       version: master
     status: developed
+  swarm_behaviors:
+    doc:
+      type: git
+      url: https://github.com/cpswarm/swarm_behaviors.git
+      version: kinetic-devel
+    release:
+      packages:
+      - position
+      - swarm_behaviors
+      - uav_local_coverage
+      - uav_optimal_coverage
+      - uav_random_direction
+      - uav_simple_tracking
+      - ugv_random_walk
+      - velocity
```

Comment:
This is also too generic.

Reply:
What about:

- swarm_behaviors_position
- swarm_behaviors_velocity